_component_design/actions-menu.md

```diff
@@ -82,28 +76,28 @@ helix-ui-javascript: true
 
 <section class="static-section" markdown="1">
 
-## Specifications
-
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Actions menu - simple
+### Specifications
 
-The most common uses of the actions menu are buttons labeled actions, split button, and cog.
+Section headers in the menu are optional. They are used if clarity is needed, or if there are more than seven actions in the list.
 
-Add a title in the action menus if it adds clarity. If there are more than seven actions in the list, the title is required.
+There are eight pixels of space between each section.
```

Comment:
I think throughout the site we've been using the numerical form with the abbreviated unit for pixel measurements. If so, recommend s/`eight pixels`/`8px`.

Reply:
Done